docs/api-irma-server.md

```diff
@@ -170,7 +171,8 @@ The `/irma` endpoints of your IRMA server have to be publicly reachable from the
 behind the `/irma` prefix are exclusively used by the
 [irmaclient](https://github.com/privacybydesign/irmago/tree/master/irmaclient)/[IRMA app](irma-app.md).
 We do not document those endpoints here, because these are only relevant for
-irmaclient developers. We only document the endpoints that are relevant for [frontend libraries](irma-frontend.md).
+irmaclient developers. We only document the endpoints that are also relevant for [frontend libraries](irma-frontend.md).
+The endpoints exclusively meant for frontend libraries can be found below [in a separate section](#api-reference-irmafrontend-endpoints).
```

Comment:
This link is broken (a dash is missing in `irmafrontend`). Additionally these two sentences are a bit double; they could perhaps be joined into one as follows:

```
However, [below](#api-reference-irma-frontend-endpoints) we document the endpoints that are also relevant for [frontend libraries](irma-frontend.md).
```

Reply:
There are a few endpoints documented there, like the `irma/session/{token}/status` endpoints. These are not exclusively for the frontend. The app uses those too. 

So, the endpoints only relevant for the app are not documented at all. The endpoints relevant both for the app and for the frontend are documented in this section and the endpoints exclusively relevant for the frontend (behind the `/frontend` prefix) are documented in the other section.

Those are in another section, because I needed some space to add the explanation about the authorization that is needed there and I couldn't add an extra header layer, because then they are ignored by the table of contents.